plugin-server/src/utils/db/db.ts

```diff
@@ -1083,6 +1083,14 @@ export class DB {
     public async queuePluginLogEntry(entry: LogEntryPayload): Promise<void> {
         const { pluginConfig, source, message, type, timestamp, instanceId } = entry
 
+        if (
+            pluginConfig.plugin?.logs_disabled &&
+            source !== PluginLogEntrySource.System &&
+            type !== PluginLogEntryType.Error
```

Comment:
Let's do logLevel for the config instead - so we have more finer control in the future if we want, allowing us to also turn on debug logging more easily while testing and otherwise have it be off.

Reply:
added a proposed loglevel solution, that can be easily expanded/iterated on though, particularly as this isn't exposed anywhere.

We can completely override this if we so wish